packages/main/src/modules/protocol.ts

```diff
@@ -6,7 +6,28 @@ import { getOSName, PLATFORM } from '../helpers';
 // Protocol Handler (deep link)
 const PROTOCOL = 'decentraland';
 
+async function handleProtocol(protocol: string | undefined) {
```

Comment:
refactor: Move the method `handleProtocol` outside the method `initProtocol` because the handlers `second-instance | open-url` can be executed before the app.whenReady trigger